src/app/_components/whereGrammersWork/WhereGrammersWrok.tsx

```diff
@@ -0,0 +1,39 @@
+import Marquee from "react-fast-marquee"
+
+const data = [
+	"Amex",
```

Comment:
Lets add an `_id` so when we fetch the data from the mongo db we could use that as a key in the map as well, thanks

Reply:
so an array of objects will be better here

Reply:
It could be like `[ {_id: '798889989', name: 'City Furniture'}, {_id: '79898987987', name: 'American Express' } ]` so that later we could add like a link or a logo etc..., thanks man

Reply:
Read your mind bro. I made the changes in the last push. Let me know if there's anything else I need to improve. 
Thanks!